PubnativeLite/PubnativeLiteDemo/SKAdNetworkViewController.h

```diff
@@ -0,0 +1,20 @@
+//
+//  SKAdNetworkViewController.h
+//  HyBid
+//
+//  Created by Orkhan Alizada on 21.09.20.
+//  Copyright © 2020 Can Soykarafakili. All rights reserved.
+//
+
+#import <StoreKit/SKStoreProductViewController.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SKAdNetworkViewController : SKStoreProductViewController {
```

Comment:
We should also prefix this class with HyBid. I'm pretty sure we'll encounter the issue where other libs have the same class.